demo/README.md

```diff
@@ -0,0 +1,56 @@
+# Demo
+
+This demo showcases a TLS-protected exchange that uses attestation evidence instead of X.509 credentials to identify and authenticate a client to a server.
+
+In [RATS terms](https://www.rfc-editor.org/rfc/rfc9334.html#figure-1), the TLS client is the attester, while the TLS server is the relying party.
+
+The relying party delegates checking the trustworthiness of the evidence presented by the attester to an attestation verifier, using a [background check](https://www.rfc-editor.org/rfc/rfc9334.html#figure-6) arrangement.
+
+A successful handshake is predicated on the provisioning of the reference values and endorsements associated with the attester to the verifier.
+
+```mermaid
```

Comment:
There is an inconsistency in the arrow names as Veraison Verifier API and Veraison Endorsement API and 
the Box as just Verifier

Either we remove the Veraison term from Arrows or we add as
Veraison Verifier

Reply:
the idea is to explicitly type the channels so I reckon there is no inconsistency here.

Reply:
There is no reference to Veraison in this page except the arrows

Reply:
Well this is not `go` isn't it. Better to have clarity...

Reply:
joking aside, none of your suggestions above seem to provide a sensible improvement over the status quo:
1. using unqualified "Verifier API" loses precision (it's a very specific interface we are using)
2. over-qualifying the verifier box loses generality

Reply:
I do not agree with point -2 above.
I did not mean you replace Verifier with Veraison

I meant for clarity you depict it as 

Verifier 
(Veraison)

The specific demo we are describing, uses a specific `Veraison` interface with a Veraison Verifier.